### ISSUE TITLE
MVMD get different result compared to VMD, is there a bug?

### ISSUE THREAD

**Issue by JeffQuantFin** [yunyueye/MVMD#1]
https://github.com/vrcarva/vmdpy/tree/master

**Comment by yunyueye** [comment]
Yes, there is a mistake in the previous mvmd.py. I have update this mvmd.py. Thank you very much!
I wrote a test program(test_mvmd.py) that you can use to test mvmd(mvmd_python.py) files.
